hmmlearn/utils.py

```diff
@@ -60,3 +60,9 @@ def iter_from_X_lengths(X, lengths):
 
         for i in range(len(lengths)):
             yield start[i], end[i]
+
+def prob_log(prob):
```

Comment:
We used to have a similar function called [`exp_mask_zero`](https://github.com/hmmlearn/hmmlearn/commit/03dd25107b940542b72513ca45ef57da22aac298), which was removed in 144dc5318b83b2385cb14239ef7b30b8e833f902, because it didn't work for non-ergodic transition matrices (see #76).